jestsetup.js

```diff
@@ -20,7 +20,5 @@ console.error = (message) => {
   }
 };
 
-window.matchMedia = () => ({ matches: true });
```

Comment:
Hmm, why remove this?

Reply:
This was moved to the jest tests that need this.